Issue #249 Adding links to generic openshift content in README

thatdocslady opened the pull request:
Fixes issue #249 

Added a few relevant links for further information on the OpenShift docs site, mainly for architecture and getting started with cluster administration.

Also reviewed using.md, and it seems that there are already some links to OpenShift docs so no need to add there.

hferentschik commented:
Add to whitelist

hferentschik commented on README.md:
```diff
@@ -5,6 +5,9 @@ cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, o
 
 Minishift uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for
 provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+For general information about OpenShift cluster architecture,  
```
Hmm, I don't think the leading paragraphs are a good place to add such a link.

hferentschik replied:
Also why "general"? Is "more" not better? What does "general" mean  in this context anyways?

Preeticp replied:
I don't think the leading paragraphs are a good place to add such a link.
+1, maybe we could add this link towards the end of the Quickstart section.

hferentschik replied:
>  maybe we could add this link towards the end of the Quickstart section.

sure.

hferentschik commented on README.md:
```diff
@@ -143,6 +146,10 @@ with the KVM hypervisor driver.
   To verify the `oc` version, check the contents of the `~/.minishift/cache/oc` directory.
 
+For general information about using the `oc` command and the command-line console,
+see [Get Started with CLI](https://docs.openshift.org/latest/cli_reference/get_started_cli.html) in
```
CLI == Command Line Interface - this does not mean anything yet. You need to specify which CLI. "Get Started with the oc CLI"

Preeticp replied:
For general information about using the oc command and the command-line console...> This should probably be - ... about using the oc command-line and the web console. 

...see Get Started with CLI in the OpenShift documentation > The link provided here mostly talks about downloading OC CLI in various Operating systems, which afaiu is exactly what Minishift does for you in the 2 steps of the Quickstart. Also, the link solely talks about OC CLI, there is no information about web console here, so we could either drop reference to the console totally or add another link for information about web console. 

My 2 cents: A more relevant link in this section would probably be 'https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#cli-reference-basic-cli-operations', which would help Developers explore OC further once Minishift launches an OC instance for them. I think that would be the logical next-step for the Dev-user.

hferentschik replied:
Actually we have the link already now - https://github.com/minishift/minishift/blob/master/docs/using.md#openshift-client-binary-oc (to the external oc doc). So I think here we should link to our docs in using.md

hferentschik replied:
> Also, the link solely talks about OC CLI, there is no information about web console here, so we could either drop reference to the console totally or add another link for information about web console.

Don't we have this as well - https://github.com/minishift/minishift/blob/master/docs/using.md#console

hferentschik commented on README.md:
```diff
@@ -174,6 +181,10 @@ To deploy the Node.js sample application from the command-line:
         Stopping local OpenShift cluster...
         Stopping "minishift"...
 
+For general information about creating applications in OpenShift,
```
I would say for "more" information.

budhrg replied:
+1 for `more`

thatdocslady commented:
@hferentschik @Preeticp updated the PR with your feedback, moved some of the links around and changed the oc link to point to our usage docs. Hopefully this resolves the concerns :)

hferentschik commented:
Rebased and merged, thanks.
